src/cmn/basicTypes.h

```diff
@@ -1,13 +1,22 @@
 /*
- * basicTypes.h
- *
- *  Created on: Jul 2, 2014
- *      Author: hariharanb
- */
+* Copyright (c) 2019 Infosys Limited
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
 
 typedef unsigned char Uint8;
 typedef unsigned short Uint16;
 typedef unsigned int Uint32;
 typedef unsigned long int Uint64;
 
-
+#include <stdbool.h>
```

Comment:
DO we need this file now ? All these defines are there in stdint.h

Reply:
Types used by the stack are Uint8 etc not from the stdint.h So we still need this file.